docs/source/cluster_management/dropwizard-bundle.rst

```diff
@@ -0,0 +1,31 @@
+.. _dropwizard-bundle:
+
+=================
+Dropwizard bundle
+=================
+
+Overview
+========
+
+The AtlasDB Dropwizard bundle can be added to Dropwizard applications. This will add startup commands to launch
+the AtlasDB :ref:`Console <console>` and :ref:`CLIs <clis>` such as ``sweep`` and ``timestamp``,
+which can be used to perform :ref:`live backups <backup-restore>`.
+
+.. warning:: These commands will only work if the server is started with a leader block in the configuration.
+
+Getting Started
+===============
+
+The Dropwizard application can be configured to use the AtlasDB Dropwizard bundle by adding
+
+``bootstrap.addBundle(new AtlasDbBundle<>());``
+
+to the ``initialize`` method.
```

Comment:
dropwizard application `initialize` method or something just to make it clear. Maybe even something like `io.dropwizard.Application#initialize` or a link to their docs.